test/redis_client_mocks/nearby_search_test.go

```diff
@@ -60,10 +76,11 @@ func TestGetPlaces(t *testing.T) {
 	// test normal cases
 	nycLatLng := POI.Location{Longitude: -74.006000, Latitude: 40.712800}
 	placeSearchRequest := iowrappers.PlaceSearchRequest{
```

Comment:
Great to see you have added more test cases. It seems that we are testing with multiple scenarios in this function, because there are different requests? Unless tests have dependencies with each other, it probably makes the future maintenance life easier if we split them into smaller tests.

Reply:
Done